components/word-of-the-day/WordOfTheDayContent.jsx

```diff
@@ -0,0 +1,60 @@
+"use client";
+import React, { useState, useEffect } from "react";
+import Loader from "../common/Loader";
+import withAuth from "@/lib/withAuth";
+import WordOfTheDayIntroduction from "@/components/word-of-the-day/WordOfTheDayIntroduction";
+import axios from "axios";
+
+const WordOfTheDayContent = () => {
+  const [word, setWordOfTheDay] = useState(null);
+
+  const fetchWordOfTheDay = async () => {
+    try {
+      const apiKey = "MYAPIKEY"; //CHANGE FOR WORDNIK API KEY
```

Comment:
instead of harcoded string, we should put process.env.WORDNIK_API_KEY here. Then I will set it as environment variable and it will work. The final line should be like this:

const apiKey = process.env.WORDNIK_API_KEY

can you correct it?

Reply:
Of course I will take a look at it. Thank you